components/sync15/Cargo.toml

```diff
@@ -6,24 +6,48 @@ authors = ["application-services@mozilla.com"]
 license = "MPL-2.0"
 exclude = ["/android", "/ios"]
 
+[features]
+# The default feature is what mozilla-central wants to make vendoring/patching easier.
```

Comment:
Could be useful to add more context here on how it's easier - like because it pulls in less dependencies etc